### PR TITLE
Parse python sources to find full strings even they were split

### PR DESCRIPTION
**Pull request by babenek**
## Description

- Added built-in parser python with ast. Python3.7 workaround applied.
- Refuzzed seeds

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] split credential string in Python code to multiple lines and check with --depth option

### CODE REVIEW

**Review by iuriimet — APPROVED** [review]
LGTM

File: credsweeper/file_handler/data_content_provider.py [hunk 1]
```diff
@@ -53,6 +54,19 @@ def represent_as_structure(self) -> bool:
             text = self.data.decode(encoding='utf-8', errors='strict')
         except Exception:
             return False
+        # # # Python
+        try:
+            if ";" in text or 2 < text.count("\n"):
```
Comment (csh519) [inline credsweeper/file_handler/data_content_provider.py:59]: I don't know why you checking `";"` in here.
Python doesn't use semicolon isn't it?

Comment (babenek) [reply credsweeper/file_handler/data_content_provider.py:59]: Semicolon may be used in python to merge source to one line. 
It is a marker that there is something complicated than HelloWorld. Imports must be separated with semicolon or new lines.
```
import urllib.request; print(urllib.request.urlopen("https://github.com/").read())
```

Comment (csh519) [reply credsweeper/file_handler/data_content_provider.py:59]: Thank you for explanation.

**Pull request merged**